appveyor.yml

```diff
@@ -35,10 +35,12 @@ for:
       - job_name: Windows-x64-debug
 
   before_build:
+  - if exist %BUILD_DIR% rmdir /S /Q %BUILD_DIR%
```

Comment:
Why are you removing the builddir? Doesn't that just get rid of all the meticulously added caching?

Reply:
To satisfy the request for building on AppVeyor, it will be removed before merge.